tests/migration/test_host_evacuate.py

```diff
@@ -20,8 +20,7 @@
 #   This network will be disconnected at some point during the tests
 
 def _host_evacuate_test(source_host, dest_host, network_uuid, vm, expect_error=False, error=""):
-    source_name = source_host.xe('host-param-get', {'uuid': source_host.uuid, 'param-name': 'name-label'})
-    vm.start(on=source_name)
+    vm.start(on=source_host.uuid)
```

Comment:
IIRC there's also a param that must be removed from above the vm.start method definition.